core/arch/arm/crypto/sub.mk

```diff
@@ -33,7 +33,10 @@ srcs-y += sm3_armv8a_ce.c
 srcs-$(CFG_ARM64_core) += sm3_armv8a_ce_a64.S
 endif
 
-ifeq ($(CFG_CRYPTO_SM4_ARM64_AESE),y)
+ifeq ($(CFG_CRYPTO_SM4_ARM_CE),y)
+srcs-$(CFG_ARM64_core) += sm4_armv8a_ce.c
+srcs-$(CFG_ARM64_core) += sm4_armv8a_ce_a64.S
+else ifeq ($(CFG_CRYPTO_SM4_ARM_AESE),y)
```

Comment:
Please do not rename `CFG_CRYPTO_SM4_ARM64_AESE` to `CFG_CRYPTO_SM4_ARM_AESE` in the second patch. Pick the name you prefer in the first patch and keep it.